static/scss/components/_books-list-view.scss

```diff
@@ -0,0 +1,72 @@
+@import "../variables";
+@import "../mixins";
+
+.list-view {
+    border: 1px solid $border-gray;
+    
+    display: grid;
+    grid-auto-rows: minmax(60px, auto);
+    
+    @include shadow-1;
+
+    a {
+        @include plain-link;
+    }
+
+    .list-header {
+        background: $backgroud-gray;
+        
+        @include list-grid($border-gray, 50px 6fr 3fr 1fr);
+
+        font-size: 0.95rem;
+        font-weight: 500;
+
+        :nth-child(2) {
```

Comment:
Can't we style the specific class name of the element you want to style here?

Reply:
We can, for some reason I've decided that this is a cleaner solution 🤔

Reply:
`nth-child` is just "dangerous" in my opinion since it introduces wrong styling when the DOM is changed (/the number or order of child elements changed).